src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Climber extends SubsystemBase {
+  private CANSparkMax climberLeftArmTwoMotor;
+  private CANSparkMax climberRightArmOneMotor;
+  private CANSparkMax climberLeftArmOneMotor;
+  private CANSparkMax climberRightArmTwoMotor;
+  private Solenoid climberSolenoidRightTilter;
+  private Solenoid climberSolenoidLeftTilter;
+  private Solenoid climberSolenoidLeftBreakOne;
+  private Solenoid climberSolenoidRightBreakOne;
+  private Solenoid climberSolenoidLeftBreakTwo;
+  private Solenoid climberSolenoidRightBreakTwo;
+
+  
```

Comment:
In addition to driving the arms at a constant speed, we also want to be able to set a desired height for the arms to go to.

This may be a later task, but here is an example of using a TrapezoidProfile to drive the position:
https://github.com/wpilibsuite/allwpilib/tree/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/elevatortrapezoidprofile

Reply:
I would start just using the position control with followers on the motoro controllers themselves however